sample_trial/services/application-dev.properties

```diff
@@ -0,0 +1,11 @@
+spring.liquibase.change-log=classpath:/db/changelog/changelog-master.xml
+spring.liquibase.drop-first=true
+spring.liquibase.enabled=true
+spring.jpa.hibernate.ddl-auto=none
+
+spring.hateoas.use-hal-as-default-json-media-type=false
+
+management.endpoints.web.exposure.include=*
+
+# Log properties in application.properties
+logging.level.uk.ac.ox.ndph.mts.trial_config_service=INFO
```

Comment:
trial_config_service is obsolete?
also, DEBUG might be better option in the dev profile.

Reply:
@darrengsmithson

Reply:
TBH the application-dev.properties was only ever meant to test the theory that it would only be used in a 'dev' profile. This property file can be deleted post merge.

Reply:
deleting